.travis.yml

```diff
@@ -30,15 +52,15 @@ cache:
   - build -> CMakeLists.txt, cmake/*
 
 before_install:
+- echo "${MATRIX_EVAL}"
 - eval "${MATRIX_EVAL}"
-- echo $CXX
-- echo $PYTHON
-- sysctl -n hw.ncpu
+- echo $TRAVIS_OS_NAME
 - pwd
+- echo $TRAVIS_BUILD_DIR
 - cd $TRAVIS_BUILD_DIR
 - mkdir taichi-llvm
 - cd taichi-llvm
-- wget https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1.zip --retry-on-http-error=403 --waitretry=3 --tries=5
+- wget https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1.zip --retry-on-http-error=403 --waitretry=3 --tries=5 || wget https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1.zip
```

Comment:
https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1.zip is for OS X.
We should use 
https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1-linux-x64.zip for Linux.

Reply:
This error is due to running an OS X executable on Linux: https://travis-ci.com/github/taichi-dev/taichi/jobs/302348630#L1817

You can fix that by using the Linux `taichi-llvm-XXX`.zip